NOTICE.txt

```diff
@@ -0,0 +1,43 @@
+
+                            The AsyncHTTPClient Project
+                            ===========================
+
+Please visit the AsyncHTTPClient web site for more information:
+
+  * https://github.com/swift-server/async-http-client
+
+Copyright 2017-2021 The AsyncHTTPClient Project
+
+The AsyncHTTPClient Project licenses this file to you under the Apache License,
+version 2.0 (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at:
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+License for the specific language governing permissions and limitations
+under the License.
+
+Also, please refer to each LICENSE.txt file, which is located in
+the 'license' directory of the distribution file, for the license terms of the
+components that this product depends on.
+
+-------------------------------------------------------------------------------
+
+
+This product contains a derivation various scripts from SwiftNIO and contains "XCTest+AsyncAwait.swift" from gRPC Swift.
```

Comment:
```suggestion
This product contains derivations of various scripts from SwiftNIO and contains "XCTest+AsyncAwait.swift" from gRPC Swift.
```

Reply:
The test derivation script from NIO is derived from somewhere else -- does the original source need to be included here too?

Reply:
Good question but I don't know.

Reply:
Yes, it does.

Reply:
I assume you mean "Tony Stone's 'process_test_files.rb'" right? If so, I have added it now.